app/tests/spec/views/sign_up.js

```diff
@@ -229,6 +248,38 @@ define(function (require, exports, module) {
             });
         });
       });
+
+      describe('password confirm', function() {
+        it('is visible if enabled', function () {
+          sinon.stub(experimentGroupingRules, 'choose', (name) => {
+            return name === 'signupPasswordConfirm' && 'treatment';
```

Comment:
@seanmonstar - I sometimes forget this was a legit way of generating a return value, I often try to do some horrible ternary with one branch returning undefined or null.